apps/sunrise/app.js

```diff
@@ -295,27 +286,19 @@ function drawClock () {
   } else {
     ypos = 32;
     fhours = 24 * (pos / w);
-    if (fhours > 23) {
-      fhours = 0;
-    }
     const nexth = 24 * 60 * (pos / w);
```

Comment:
This and the next line can be replaced with `fmins = 59 - ((24 * 60 * (1 - (pos / w))) % 60);`, but I didn't include it in the request, because the readability isn't great.